packages/tools/stylelint-config-pie/rules/strict.js

```diff
@@ -1,8 +1,7 @@
-module.exports = {
+export default {
     rules: {
         'color-named': 'never',
         'max-nesting-depth': 3,
-        'media-query-list-comma-newline-after': 'always-multi-line',
```

Comment:
With all these rules that are being removed, are they deprecated or are they now just the default values or something?

Reply:
they are deprecated https://stylelint.io/migration-guide/to-15/#deprecated-stylistic-rules

Stylelint is offloading this to prettier and Eslint as mentioned in their guide

> 
> When we created these rules, pretty printers (like [Prettier](https://prettier.io/)) didn't exist. They now offer a better way to consistently format code, especially whitespace. Linters and pretty printers are complementary tools that work together to help you write consistent and error-free code.
> 
> By deprecating these rules, we can:
> 
> focus on writing and maintaining rules that help you [avoid errors](https://stylelint.io/user-guide/rules#avoid-errors) and [enforce (non-stylistic) conventions](https://stylelint.io/user-guide/rules#enforce-conventions), both of which are unique to Stylelint
> modernize our codebase, e.g. move to ESM so that we can update our dependencies and keep Stylelint secure for you
>